index.js

```diff
@@ -92,9 +91,10 @@ const kickOffAction = async (inputJson) => {
     Object.entries(utils.populateMap(PRIORITY_MAPPER))
   );
   const reportPairsMapper = utils.populateMap(REPORT_INPUT_KEYS);
+  const issueLabelsMapper = utils.getInput('ISSUE_LABELS_MAPPER');
   const labels =
-    ISSUE_LABELS_MAPPER.length !== 0
-      ? { labels: ISSUE_LABELS_MAPPER.split(',') }
```

Comment:
What if length is exactly === 1? Doesn't if matter to Jira to have one entry in the labels list in the query with a trailing comma?

Reply:
Yep, this is a valid point. Will address.